Makefile

```diff
@@ -155,13 +155,23 @@ $(PYMODEL):
 $(DOCDIR):
 	mkdir -p $@
 
+# Compile static Markdown files, images, and JavaScript scripts, into a documentation website.
+#
+# Then, use `refgraph` (part of `refscan`) to generate a pair of graphs (i.e. network diagrams),
+# one that depicts inter-collection relationships and one that depicts inter-class relationships.
```

Comment:
My two cents here is it will be confusing and not very useful to distribute the inter-class relationship diagram until we make the ranges as strict as the structured syntax patterns.